src/communication/WebSocketBridgeCommunication.ts

```diff
@@ -66,40 +65,40 @@ export class WebSocketBridgeCommunication {
     public getXcApiList(): Promise<Array<string>> {
         const thisWebSocketBridgeCommunication = this;
         const command = Commands[Commands.getXcApiList];
-        const promise = this.updates$
-            .filter((data: DeserializedData) => data.command === command)
-            .first()
-            .map((data: DeserializedData) => {
+        const promise = this.updates$.pipe(
+            filter((data: DeserializedData) => data.command === command),
+            first(),
+            map((data: DeserializedData) => {
                 this.logger.info("ApiList received successfully");
                 return thisWebSocketBridgeCommunication.deserializer.getJsonDataFromGetXcApiListRequest(data.stringData);
-            })
+            }), )
```

Comment:
ugly

Reply:
Sorry, the auto migration tool did this. I will fix it.

Reply:
Fixed